coverage/index.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
```

Comment:
Please add `/coverage/*` to your `.gitignore` file and run `git rm coverage/index.html` and so on for each of the files showing up here under `coverage` folder